packages/main/test/specs/base/AriaLabelHelper.spec.js

```diff
@@ -0,0 +1,137 @@
+const assert = require('chai').assert;
+
+describe('AriaLabelHelper', () => {
+  before(async () => {
+    await browser.url(`test/pages/base/AriaLabelHelper.html`);
+  });
+
+  const getErrorMessageAriaLabelNotAsExpected = (actual, expected) => {
+    return `aria-label ${actual} is not as expected ${expected}.`;
+  };
+
+  const testInputAriaLabelMatchesLabels = async (inputId, labelIds) => {
+    const input = await browser.$(`#${inputId}`);
+    const innerInput = await input.shadow$('input');
+    const actualAriaLabel = await innerInput.getAttribute('aria-label');
+
+    const promises = labelIds.map(async (labelId) => {
+      const label = await browser.$(`#${labelId}`);
+      return await label.getText();
+    });
+    const texts = await Promise.all(promises);
+    const expectedAriaLabel = texts.join(' ');
+
+    assert.equal(
+      actualAriaLabel,
+      expectedAriaLabel,
+      getErrorMessageAriaLabelNotAsExpected(actualAriaLabel, expectedAriaLabel)
```

Comment:
Don't we need to return a positive message here not getErrorMessageAriaLabelNotAsExpected?